go.mod

```diff
@@ -7,7 +7,6 @@ require (
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/Microsoft/go-winio v0.5.2
 	github.com/ProtonMail/go-crypto v0.0.0-20220623141421-5afb4c282135
-	github.com/RedHatQE/gowinx v0.0.3
```

Comment:
Is it a step back from our automation effort? You folks effort to put the UX test as part of automation and now we are removing those because of it is outdated or not works as expected or have some kind of limitation? Can it be fixed instead removing the functionality?

Reply:
@adrianriobo you can have it here but in the commit message put those intentions which allow us (reviewers) to understand why it is done.

Reply:
done